java-spiffe-core/src/test/java/io/spiffe/workloadapi/FakeWorkloadApi.java

```diff
@@ -66,11 +66,23 @@ public void fetchX509SVID(Workload.X509SVIDRequest request, StreamObserver<Workl
                     .setX509Svid(svidByteString)
                     .setX509SvidKey(keyByteString)
                     .setBundle(bundleByteString)
+                    .setHint("external")
+                    .build();
+
+            // This svid should be filtered out by the client because it has a non-unique hint.
```

Comment:
```suggestion
            // This X.509-SVID should be filtered out by the client because it has a non-unique hint and is not the first X.509-SVID in the response with this hint.
```